README.md

```diff
@@ -1,2 +1,23 @@
 # HW01
-Edit the readme
+## Xingruo (Summer) Zhang
+
+<img src="https://github.com/zhangxingruo/HW01/blob/master/Xingruo%20Zhang.png" width = "40%">
+
+*Education*: I recently graduated from UCLA with majors in Statistics and Economics. I will be joinng UChicago for the Public Health Sciences PhD program this fall.  
+
+*Interests*: Below would be a list of things that excite me.  
+
+* fluffy animals  
+    + especially puppies 
+    + [https://images.app.goo.gl/QdoGfMaLTm353wdaA]
+
+* chocolate & ice-cream  
+* yoga, kayaking, & Marathons  
```

Comment:
I also enjoy running. I hope I can finish my first Marathon in the next five years...